tests/app/test_users.py

```diff
@@ -181,6 +190,20 @@ def test_return_400_for_invalid_user_json(self):
                 'users': {
                     'email_address': 'joeblogs@email.com',
                     'password': '',
+                    'role': 'buyer',
+                    'name': 'joe bloggs'}}),
+            content_type='application/json')
+
+        assert_equal(response.status_code, 400)
```

Comment:
Stuck an extra clause to check response. At this point the responses are very uninformative - simply JSON not valid. We should think more broadly about our responses so that they inform of the nature of JSON errors.